templates/mrepo.conf.erb

```diff
@@ -18,6 +18,12 @@ rhnlogin = <%= @rhn_username %>:<%= @rhn_password %>
 rhnlogin = <%= @rhn_username %>
 <% end -%>
 
-<% unless (mailto = @mailto) == 'UNSET' -%>
-mailto = <%= mailto %>
+<% if @mailto -%>
+mailto = <%= @mailto %>
+<%   if @mailfrom -%>
```

Comment:
The indented `<%   if @mailfrom -%>` is to indicate that that is inside the surrounding `<% if @mailto -%>` block.  The `mailfrom` and `smtp-server` parameters only make sense if `mailto` is set.

I could indent the `<%` itself, but I'd have to transform it to <code>&nbsp; &lt;%-</code>, and that also looks strange when the parameters inside that `if` block are not themselves indented.